edx_proctoring/templates/proctoring/seq_proctored_exam_ready_to_start.html

```diff
@@ -0,0 +1,62 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Your Proctoring Installation and Set Up is Complete
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Placeholder content. <strong>Please keep the proctoring software open and running while you complete your exam</strong>.
```

Comment:
"Do not close the proctoring session window until you have completed and submitted your exam."